### ISSUE TITLE
关于设备DID，还有测试服务器

### ISSUE THREAD

**Issue by jjiikkkk** [wxtcstt/MiChatGUI#2]
其他人的代码形式，大多数人小米ID和密码都通不过验证，这样的话，我们就看不到DID
还有其他方式可以看到DID吗

另外测试服务器，是不是要代理电脑上做全局啊

**Comment by wxtcstt** [comment]
是的 ，代理电脑上做全局。有一种方式不需要DID，另一种方式需要DID，你更新一下代码，我把两种方式在界面上做了选择，有一种方式自动获取DID

**Comment by jjiikkkk** [comment]
你的软件我用了，不要DID的，可能代理设置错误了，效果暂时不太好，我再试试换个代理
要DID的，他们的代码基本账号和密码都过不去验证，你的小米账号和密码可以吗
是大量的人都过不去验证，我们从国外云服务器换到了国内云服务器

**Comment by jjiikkkk** [comment]
本地全局估计速度够不上，我还是远程装windows上去用
